### PR TITLE
docs: add contributing info and code of conduct

### PR DESCRIPTION
**Pull request by bohmber**
The contributing info contains a suggestion for a commit message format. Please comment

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* **fix**: A bug fix
+* **docs**: Documentation only changes
+* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
+  semi-colons, etc)
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **perf**: A code change that improves performance
+* **test**: Adding missing tests
+* **chore**: Changes to the auxiliary tools such as release scripts
+* **build**: Changes to the dependencies, devDependencies, or build tooling
+* **ci**: Changes to our Continuous Integration configuration
```
Comment (henningn) [inline CONTRIBUTING.md:38]: Do we need so many different type types? 
"perf" seems to be a subtype of "refactor".
"build" and "ci" may be a subtype of "chore".

So I suggest to remove those subtypes.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+##### Scope
+
+The scope could be anything that helps specify the scope (or feature) that is changing.
```
Comment (henningn) [inline CONTRIBUTING.md:42]: Should the scope be optional?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+##### Body
+
+Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
+The body should include the motivation for the change and contrast this with previous behavior.
+
+##### Footer
+
+The footer should contain any information about **Breaking Changes** and is also the place to
+reference Jira issues that this commit **Closes**, **Fixes**, or **Relates to**.
```
Comment (henningn) [inline CONTRIBUTING.md:64]: Do we need "Closes, Fixes, Relates to" for referencing an Issue?
We could use "Issue: TOBAGO-1234" instead.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bohmber) [reply CONTRIBUTING.md:38]: I don't like 'chore' it's like 'item'. To be a little bit more specific should be helpful if you are searching for a problem.
'chore' should be avoided if you have a better type.

File: CONTRIBUTING.md (hunk 3 above)
Comment (bohmber) [reply CONTRIBUTING.md:64]: Only using 'Issue: TOBAGO-XXXX' is fine for me

File: CONTRIBUTING.md (hunk 2 above)
Comment (bohmber) [reply CONTRIBUTING.md:42]: Yes scope is optional. I will change the text a little bit

File: CONTRIBUTING.md (hunk 1 above)
Comment (henningn) [reply CONTRIBUTING.md:38]: Ok, good point. Then we should keep "ci" and "build"
What about "perf"? I belive "perf" would be used very rarely. I can only remember one commit a few years ago where this type could have been applied.

Comment (lofwyr14) [reply CONTRIBUTING.md:38]: I suggest a type **demo** additional to **docs** (it's not the same)
I'm not sure, but **vm* might be useful for the tobago-vm.apache.org stuff.
What about version changes as release preparations? The Jira versions are managed in demo, so we can use **demo**.
Or is **demo** a scope?

**Review by lofwyr14 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+* no period (.) at the end
+
+##### Body
+
+Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
+The body should include the motivation for the change and contrast this with previous behavior.
+
+##### Footer
+
+The footer should contain any information about **Breaking Changes** and is also the place to
```
Comment (henningn) [inline CONTRIBUTING.md:63]: Is the term "breaking changes" used only to indicate, that the main version number must be increased?
Or is it also used for internal breaking changes, like the change from QUnit to Jasmine?

File: CONTRIBUTING.md (hunk 1 above)
Comment (henningn) [reply CONTRIBUTING.md:38]: If you fix an example in the demo, do you write the commit title?
* "demo(popup): fix example"
* "fix(demo): popup example"

File: CONTRIBUTING.md (hunk 4 above)
Comment (bohmber) [reply CONTRIBUTING.md:63]: Breaking change for the user.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bohmber) [reply CONTRIBUTING.md:38]: I'm fine with "fix(demo): popup example". "demo" is more a scope.

**Pull request merged**